### PR TITLE
chore: run tests on linux and windows

### LINKED ISSUE #227: Automatically run integration tests on Windows & Linux
Automatically run integration tests on Windows & Linux to ensure both are supported properly.

### PR DESCRIPTION
**Pull request by stijnmoreels**
Change build/release pipeline to run (by default) the unit/integration tests on both Linux and Windows build machines.

Closes #227

### CODE REVIEW

File: build/ci-build.yml [hunk 1]
```diff
@@ -93,8 +93,15 @@ stages:
     jobs:
       - job: RunIntegrationTests
         displayName: 'Run integration tests'
+        strategy:
+          maxParallel: 1
+          matrix:
+            linux:
+              imageName: 'ubuntu-latest'
+            windows:
+              imageName: 'windows-latest'
```
Comment (stijnmoreels) [inline build/ci-build.yml:102]: Apparently, we can't use variables in strategy.matrix variable definitions... too bad.

Comment (fgheysels) [reply build/ci-build.yml:102]: What exactly do you mean ?  Do you mean you want to refer to the `Vm.Linux.Image` and `Vm.Windows.Image` variables ?

Is this also not possible via [Template Expression Syntax](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#template-expression-syntax) ? Like this:

```yaml
${{ variables['Vm.Linux.Image']}}
```

Comment (stijnmoreels) [reply build/ci-build.yml:102]: I'll try that, but don't have much hope 😅 .

Comment (stijnmoreels) [reply build/ci-build.yml]: Aha, yeah, that did it. Forgot that template expression variables were initialized at runtime, and would therefor do the trick. Thx!

**Review by fgheysels — APPROVED** [review]

File: src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.Sql.tests.ps1 [hunk 2]
```diff
@@ -99,7 +99,7 @@ InModuleScope Arcus.Scripting.Sql {
             # avoid having timeout errors during the test themselves.
             try {
                 Write-Host "Execute dummy SQL statement to make sure the Azure SQL DB is resumed."
-                Invoke-Sqlcmd @params -Query "SELECT TOP 1 FROM INFORMATION_SCHEMA.TABLES" -ConnectionTimeout 60 -Verbose
+                Invoke-Sqlcmd @params -Query "SELECT TOP 1 FROM INFORMATION_SCHEMA.TABLES" -ConnectionTimeout 60 -Verbose -ErrorAction SilentlyContinue
```
Comment (fgheysels) [inline src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.Sql.tests.ps1:102]: Do we still need the try/catch statement if we specify `SilentlyContinue`?

Comment (stijnmoreels) [reply src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.Sql.tests.ps1:102]: Probably not 😉 .

Comment (stijnmoreels) [reply src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.Sql.tests.ps1]: Changed it in https://github.com/arcus-azure/arcus.scripting/pull/233/commits/fb24a476f1cdc49a93579b17558ce08eeffadc82

Comment (stijnmoreels) [reply src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.Sql.tests.ps1]: Had to fix the SQL statement, as there was missing a selection.

**Comment by stijnmoreels** [comment]
> I think this is OK for me, although I don't see what has been changed on that sql statement :?

There was a missing column selection in the statement. I added `TABLE_NAME` to have something.

**Comment by stijnmoreels** [comment]
Woops, was merged without @pim-simons 's consent. Let me know if there is anything you would want to change!

**Pull request merged**